lib/index.ts

```diff
@@ -214,6 +222,15 @@ export function isG1Rfc9380(value: any, info: ChainInfo): value is G1RFC9380Beac
         value.round > 0
 }
 
+// eslint-disable-next-line  @typescript-eslint/no-explicit-any
+export function isBn254OnG1(value: any, info: ChainInfo): value is Bn254OnG1Beacon {
+    return info.schemeID === 'bls-bn254-unchained-on-g1' &&
+        !!value.randomness &&
```

Comment:
At some point it'd be nice to start using the v2 APIs and in these the randomness value isn't there anymore, instead it is computed by the client.